### PR TITLE
Add error icon

### PR DESCRIPTION
**Pull request by expe**

### CODE REVIEW

File: scss/_ft-icons.scss [hunk 1]
```diff
@@ -1,124 +1,126 @@
 // DO NOT EDIT. This file is generated by grunt-webfont
-$_o-ft-icons-font-file-name: str-slice('url("../build/ft-icons-f414da96fb9364a6326bee01eb65edef.eot")', 9, -7);
+$_o-ft-icons-font-file-name: str-slice('url("../build/ft-icons-5f92b931e2324d43c30f91f35d9c7242.eot")', 9, -7);
 
 
 // Define character code for each icon
 $_o-ft-icons-icon-map: (
 
-
+
```
Comment (AlbertoElias) [inline scss/_ft-icons.scss]: Is it adding tabs?

File: svg/error.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="83" height="81.167" viewBox="0 0 83 81.167"><path d="M42.52 53.824c-1.996 0-3.606 1.612-3.606 3.608 0 2 1.606 3.612 3.607 3.612 2.006 0 3.606-1.613 3.606-3.612 0-1.995-1.6-3.608-3.605-3.608zM42.52 26.24c-2.163 0-3.665 2.06-3.665 3.948.612 3.89 2.833 17.077 3.28 20.005h.777c.443-2.928 2.662-16.102 3.277-19.988 0-1.89-1.442-3.965-3.67-3.965z"/><path d="M74.704 59.318L48.48 13.9c-1.64-2.84-3.798-4.258-5.96-4.258-2.16 0-4.32 1.418-5.958 4.258L10.34 59.318C7.06 64.995 9.743 69.64 16.3 69.64h52.445c6.554.002 9.237-4.645 5.96-10.322zm-3.62 4.786c-.08.105-.747.537-2.34.537H16.3c-1.594 0-2.26-.43-2.336-.528-.052-.12-.09-.914.705-2.294L40.892 16.4c.797-1.38 1.502-1.742 1.623-1.758.133.016.84.38 1.635 1.758l26.225 45.418c.796 1.38.757 2.173.71 2.286z"/></svg>
```
Comment (AlbertoElias) [inline svg/error.svg]: It's actually a "warning" icon

**Comment by expe** [comment]
Renamed the icon to warning and fixed tabs in css file. Unsure why it started adding tabs to the generated file.

**Comment by AlbertoElias** [comment]
:+1: feel free to ship it

**Pull request merged**